app/src/main/java/com/portfolio/tasky/entry/fragments/RegistrationFragment.kt

```diff
@@ -40,7 +44,17 @@ class RegistrationFragment : Fragment(), FragmentInflater by FragmentInflaterImp
 
         setObservers()
         viewBinding.btnReg.setOnClickListener {
-            startLoginFragment()
+            val name = viewBinding.etName.subLayout.etInput.text.toString()
+            val email = viewBinding.etEmail.subLayout.etInput.text.toString()
+            val password = viewBinding.etPassword.subLayout.etInput.text.toString()
+            viewModel.makeRegistrationCall(RegisterRequest(name, email, password)).observe(viewLifecycleOwner){
+                    it.let {  if(it == true){
+                        Toast.makeText(activity, "Success", Toast.LENGTH_SHORT).show()
+                    }else{
+                        Toast.makeText(activity, "Failure", Toast.LENGTH_SHORT).show()
+                    }
```

Comment:
Here you could just
```
if(it == true){
    Toast.makeText(activity, "Success", Toast.LENGTH_SHORT).show()
}else{
    Toast.makeText(activity, "Failure", Toast.LENGTH_SHORT).show()
}
```

Avoid the unnecesary .let in this case

Reply:
Done. Current implementation checks for a non-null value, if present then only Toast is called.